pkg/certificate/manager.go

```diff
@@ -152,9 +154,83 @@ func (m *Manager) handleMRCEvent(mrcClient MRCClient, event MRCEvent) error {
 			m.mu.Unlock()
 		}
 	case MRCEventUpdated:
-		// TODO
+		oldMRC := event.OldMRC
+		newMRC := event.NewMRC
+
+		// check if the MRC's intent has been updated from active to passive and the
+		// certificate statuses are all issuing
+		if oldMRC.Intent != newMRC.Intent && newMRC.Intent == constants.MRCIntentPassive &&
+			componentsAreIssuing(newMRC) {
+			log.Debug().Str("mrc", newMRC.Name).Msg("handling active to passive MRC status change")
+			err := handleActiveToPassiveMRCStatusChange(mrcClient, event)
+			if err != nil {
+				return err
+			}
+
+			// define reconciler
+			mrcReconciler := MRCReconciler{
+				mrcName:      newMRC.Name,
+				updateStatus: nil,
```

Comment:
can you add a comment explaining what the updateStatus might do in this example?

Reply:
Sure! As outlined in #5091:
> Once the condition is met, set the `status.condition.IssuingRollback` to `False` and the reason to `RollbackComplete`. Also, set the `status.state` to `Validating`